src/web-map.js

```diff
@@ -211,6 +234,9 @@ export class WebMap extends HTMLMapElement {
     let output = "<output role='status' aria-live='polite' aria-atomic='true' class='mapml-screen-reader-output'></output>";
     this._container.insertAdjacentHTML("beforeend", output);
 
+    let outputScale = "<output role='status' aria-live='polite' aria-atomic='true' class='mapml-screen-reader-output-scale'></output>";
```

Comment:
Yes, I believe it would be more organized and also easier to debug (future purposes) for scale bar to have its own output element. As well, adding text onto another element with already existing text might cause timing issues related to NVDA.